gradoop-examples/src/main/java/org/gradoop/benchmark/sna/SocialNetworkAnalyticsExample.java

```diff
@@ -165,7 +164,9 @@ private static void readCMDArguments(CommandLine cmd) {
     INPUT_PATH      = cmd.getOptionValue(OPTION_INPUT_PATH);
     OUTPUT_PATH     = cmd.getOptionValue(OPTION_OUTPUT_PATH);
     CSV_PATH        = cmd.getOptionValue(OPTION_CSV_PATH);
-    STATISTICS_PATH = cmd.getOptionValue(OPTION_STATISTICS_PATH);
+    if (cmd.hasOption(OPTION_STATISTICS_PATH)) {
```

Comment:
I dont understand this change. The sanity of `OPTION_STATISTICS_PATH` is checked in the `performSanityCheck` function. Why would you remove the sanity check there and add it here?

Reply:
Ah i see. You mean that this option is optional. You just need to remove the sanity check for that. Right now you will receive `null` if the option is not set.